Documentation/Configuration/TypoScript/Syntax/Conditions/Index.rst

```diff
@@ -0,0 +1,150 @@
+evaluates to true. Examples of condition criteria are:
+
+-  Is a user logged in?
+
+-  Is it Monday?
+
+-  Is the page called in a certain language?
+
+Conditions are a TypoScript syntax construct. They are thus available in both
+frontend TypoScript and backend TSconfig. However, condition criteria are based
+on prepared variables and functions, and those are different in frontend
+TypoScript and backend TSconfig. For example, the :typoscript:`frontend` variable does
+not exist in TSconfig, it is (obviously) impossible to have a backend TSconfig
+condition that checks for a logged in frontend user.
+
+For a reference of allowed condition criteria, please refer to the according
+chapter in the :ref:`frontend TypoScript Reference <t3tsref:conditions>` and
+the :ref:`backend TSconfig Reference <t3tsconfig:conditions>`. These references
+come with examples for single condition criteria as well.
+
+The Tsconfig and TypoScript backend modules show lists of existing conditions
+and allow simulating criteria verdicts to to analyze their impact on the
+resulting TypoScript tree.
+
+Condition criteria are based on
+`symfony expression language <https://symfony.com/doc/current/components/expression_language/syntax.html>`__.
+The core allows extending the symfony expression language with own variables and
+functions, see :ref:`symfony expression language API <sel-within-typoscript-conditions>`
+for more details.
+
+
+Basic example
+=============
+
+.. include:: /CodeSnippets/TypoScriptSyntax/Conditions1.rst.txt
+
+
+.. _typoscript-syntax-conditions-syntax:
+
+Syntax and rules
+================
+
+The general syntax is like this:
+
+.. include:: /CodeSnippets/TypoScriptSyntax/Conditions2.rst.txt
+
+These general rules apply:
+
+* Conditions are encapsulated in :typoscript:`[` and :typoscript:`]`
+
+* :typoscript:`[ELSE]` negates a previous condition criteria and can contain
+  a new body until :typoscript:`[END]` or :typoscript:`[GLOBAL]`. :typoscript:`[ELSE]`
+  is considered if the condition criteria did *not* evaluate to true.
+
+* :typoscript:`[END]` and :typoscript:`[GLOBAL]` stop a given condition scope.
+  This is similar to a closing curly brace :code:`}` in programming languages like PHP.
+
+* .. versionchanged:: 12.0
+
+  :typoscript:`[END]` and :typoscript:`[GLOBAL]` behave exactly the same. Both
+  are kept for historical reasons (for now).
+
+* Conditions can use constants. They are available in frontend TypoScript "setup" and
+  in TSconfig from "site settings. A simple example if this constant
+  :typoscript:`myPageUid = 42` is set:
+
+  .. code-block:: typoscript
+
+      [traverse(page, "uid") == {$myPageUid}]
+          page.10.value = Page uid is 42
+      [end]
+
+* Multiple condition criteria can be combined using :typoscript:`or` or :typoscript:`||`,
+  as well as :typoscript:`and` or :typoscript:`&&`
+
+* Single criteria can be negated using :typoscript:`!`
+
+* Conditions can *not* be nested within code blocks.
```

Comment:
If it never changed they can't be nested in other context either.
So this example from above never states anything about  A below [if B]:

```
[if A]
    do something related to A
[if B]
    do something related to B
[END]
```